src/SharpRaven.sln

```diff
@@ -191,6 +209,12 @@ Global
 		{7FE52A2B-430C-4C6F-BEA9-0855AF973D0C}.Debug 4.5|Mixed Platforms.ActiveCfg = Debug 4.5|Any CPU
 		{7FE52A2B-430C-4C6F-BEA9-0855AF973D0C}.Debug 4.5|Mixed Platforms.Build.0 = Debug 4.5|Any CPU
 		{7FE52A2B-430C-4C6F-BEA9-0855AF973D0C}.Debug 4.5|x86.ActiveCfg = Debug 4.5|Any CPU
+		{7FE52A2B-430C-4C6F-BEA9-0855AF973D0C}.Release 3.5|Any CPU.ActiveCfg = Release 4.5|Any CPU
+		{7FE52A2B-430C-4C6F-BEA9-0855AF973D0C}.Release 3.5|Any CPU.Build.0 = Release 4.5|Any CPU
+		{7FE52A2B-430C-4C6F-BEA9-0855AF973D0C}.Release 3.5|Mixed Platforms.ActiveCfg = Release 4.5|Any CPU
+		{7FE52A2B-430C-4C6F-BEA9-0855AF973D0C}.Release 3.5|Mixed Platforms.Build.0 = Release 4.5|Any CPU
+		{7FE52A2B-430C-4C6F-BEA9-0855AF973D0C}.Release 3.5|x86.ActiveCfg = Release 4.5|Any CPU
+		{7FE52A2B-430C-4C6F-BEA9-0855AF973D0C}.Release 3.5|x86.Build.0 = Release 4.5|Any CPU
```

Comment:
I think by simply opening and saving the solution it made those changes. I did not modify anything knowingly.

I might not have 3.5 installed, perhaps that is why.